quirks/change-password-URLs.json

```diff
@@ -91,5 +92,5 @@
     "yahoo.com": "https://login.yahoo.com/account/change-password",
     "zhihu.com": "https://www.zhihu.com/settings/account",
     "zocdoc.com": "https://www.zocdoc.com/patient/editprofile?section=Password",
-    "zoom.us": "https://zoom.us/profile#pwd-form"
+    "zoom.us": "https://zoom.us/profile#pwd-form",
```

Comment:
A trailing comma is not valid `JSON` unfortunately, would you mind removing it please?

Reply:
Sure! Done.